packages/grid/docker-compose.multinode.yml

```diff
@@ -0,0 +1,461 @@
+version: "3.8"
+services:
+  # docker-host:
+  #   image: qoomon/docker-host
+  #   cap_add:
+  #     - net_admin
+  #     - net_raw
+
+  proxy:
+    restart: always
+    hostname: ${NODE_NAME?Variable not set}
+    image: ${DOCKER_IMAGE_TRAEFIK?Variable not set}:${TRAEFIK_VERSION?Variable not set}
+    networks:
+      - "${TRAEFIK_PUBLIC_NETWORK?Variable not set}"
+      - default
+    volumes:
+      - "./traefik/docker/traefik.yml:/etc/traefik/traefik.yml"
+      - "./traefik/docker/dynamic.yml:/etc/traefik/conf/dynamic.yml"
+    environment:
+      - SERVICE_NAME=proxy
+      - RELEASE=${RELEASE:-production}
+      - HOSTNAME=${NODE_NAME?Variable not set}
+      - HTTP_PORT=${HTTP_PORT}
+      - HTTPS_PORT=${HTTPS_PORT}
+    ports:
+      - "${HTTP_PORT}:81"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    labels:
+      - "orgs.openmined.syft=this is a syft proxy container"
+
+    # depends_on:
+    #   - "docker-host"
+
+  tailscale:
```

Comment:
@shubham3121 @koenvanderveen  we have removed tailscale on "dev". perhaps we have to sync this file against dev as well. Please let me know if we still need tailscale for multinode setup.

Reply:
this file can be removed probably